php/simps/swoole.ini

```diff
@@ -0,0 +1,3 @@
+[swoole]
```

Comment:
Could you remove this file ?
It's not needed

Reply:
I need to turn off enable_library because the composer of `swoole/library` conflicts with the underlying swoole.
Or I modified the command to `php -d swoole.enable_library=off sbin/simps.php http: start`?

Reply:
Oh, i see `before_command`, Let me delete it